lib/node_modules/@stdlib/error/tools/database/data/data.csv

```diff
@@ -908,3 +908,4 @@
 "Ed","invalid argument. Must provide a two-dimensional ndarray. Value: `%s`.","TypeError"
 "Ee","invalid argument. Number of matrix columns must match centroid dimensions. Expected: `%u`. Actual: `%u`.","Error"
 
+
```

Comment:
@Planeshifter The workflow seems to be adding additional newlines to the end of the CSV file. There should be only a single trailing newline.

Reply:
@kgryte Thanks for flagging! Updated the script to address this.